app/assets/stylesheets/dark-mode.css

```diff
@@ -134,3 +134,11 @@ pre:not(.line-numbers) code {
 .advert__icon {
   filter: invert(1);
 }
+
+.odin-dark-clear-button {
+  color: var(--dark-header-color) !important;
+}
+
+.odin-dark-clear-button:hover {
+  color: black !important;
+}
```

Comment:
Yeah, they are needed unfortunately. Our dark mode styles were basiclly copied and pasted into the app from the chrome extension we had before. They are loaded from a different stylesheet tag and need to override a lot of the light mode styles which opens many specificity issues. 

But this will all be removed and replaced with[ Tailwinds dark mode system](https://v2.tailwindcss.com/docs/dark-mode) once we move all the pages over. Definitely consider this file a temporary thing 😄